### ISSUE TITLE
Nova Branch

### ISSUE THREAD

**Issue by prinakayama** [wesleygomes022/GH4Woman-modulo1#10]
Nao consegui criar nova branch pq diz que nao tenho acesso

**Comment by jaquedreyer** [comment]
Também não recebi ainda o convite para colaborar no repositório @wesleygomes022

**Comment by monicacruzs** [comment]
Favoritei para poder executar o ultimo comando @wesleygomes022

**Comment by cassucena** [comment]
Mesmo Problemas por aqui... nao tenho permissão para fazer o push.

**Comment by pilardib** [comment]
eu também estou aguardando a permissão para conseguir executar o último passo

**Comment by nayaraosorio** [comment]
tentei também, fala que não tenho permissão..

ERROR: Permission to GH4WOMAN-M1/GH4Woman-modulo1.git denied to nayaraosorio.
fatal: Could not read from remote repository.

**Comment by Jhennyf** [comment]
Infelizmente, não estou conseguindo também!

**Comment by Shyuu7** [comment]
Também estou aguardando a permissão de acesso

**Comment by DaianeCP** [comment]
Eu tb, só no aguardo....

**Comment by beatrizalsilva** [comment]
Não consegui criar uma nova branch porque diz que não tenho acesso. Mandei um email falando sobre o erro, e estou aguardando permissão. 
![Captura de tela 2024-05-17 191454](https://github.com/GH4WOMAN-M1/GH4Woman-modulo1/assets/146834458/f99bd132-a75a-4d6d-965b-63ca109435ca)

**Comment by lanvsc** [comment]
O mesmo aqui.

**Comment by velfortrab** [comment]
<img width="656" alt="erro-push" src="https://github.com/GH4WOMAN-M1/GH4Woman-modulo1/assets/88948434/2cf53be9-344c-40bb-bc2d-cec932244bef">

**Comment by janacris** [comment]
Segue novamente o erro de permissão:

![image](https://github.com/GH4WOMAN-M1/GH4Woman-modulo1/assets/89252310/8e405702-2a45-405e-a8b5-7d3bd5385de5)

**Comment by adelitab** [comment]
Estou com o mesmo problema desde ontem na aula ao vivo:

PS C:\Users\joaop\Desktop\GH4Woman-modulo1> git push -u origin abarriquelo
info: please complete authentication in your browser...
remote: Permission to GH4WOMAN-M1/GH4Woman-modulo1.git denied to adelitab.
fatal: unable to access 'https://github.com/GH4WOMAN-M1/GH4Woman-modulo1.git/': The requested URL returned error: 403

@wesleygomes022 nos ajude por favor!

**Comment by fabricianalima** [comment]
Estou com o mesmo problema @wesleygomes022, não consigo fazer o push!

**Comment by SabrinaFurtado** [comment]
tem como me enviar o convite por gentileza @wesleygomes022

**Comment by marinabagliano** [comment]
também estou com o mesmo problema, preciso do convite por favor, obrigada!!

**Comment by jaq442** [comment]
também estou com o mesmo problema! @wesleygomes022

**Comment by janacris** [comment]
O meu funcionou depois que eu aceitei o convite dele.No meu caso, era esse o problema.Daí consegui terminar o exercício normalmente depois.

**Comment by juliakarla** [comment]
Não recebi nenhum convite!
Eles tem q enviar pra mim?

Em sáb., 18 de mai. de 2024 23:33, Janaíne Assis ***@***.***>
escreveu:

**Comment by janacris** [comment]
@juliakarla pelo que eu entendi sim porque você precisa de permissão para fazer alterações no repositório da organização. Lembra que @wesleygomes022  tinha dito, na noite da aula pra gente não fazer o exercício ainda, porque ele precisava "liberar" a gente, pois se não, não iríamos conseguir também? Acho que era disso que ele estava falando...

**Comment by juliakarla** [comment]
Sim! Eu lembro! Era pra clicar na estrelinha do lado direito. Fiz isso
durante a aula qd ele ensinou. Ainda assim o erro persiste!

Em sáb., 18 de mai. de 2024 23:45, Janaíne Assis ***@***.***>
escreveu:

**Comment by BenedettiMih92** [comment]
também estou com o mesmo problema! @wesleygomes022

**Comment by SabrinaFurtado** [comment]
Meninas, eu consegui mas somente pelo VS Code, tentem por lá que vai.

**Comment by pilardib** [comment]
eu tentei pelo vs code e mesmo assim dá problema. acho que não tenho permissão ainda

**Comment by camilaccb** [comment]
@wesleygomes022 consegue me enviar o convite para o repositório? Não estou conseguindo concluir o comando de push

**Comment by juliakarla** [comment]
Fiz tudo do zero pelo VS Code e dá o mesmo erro!

Em dom., 19 de mai. de 2024 às 18:34, Camila Caldas <
***@***.***> escreveu:

**Comment by TalitaMaziero** [comment]
Eu  também, sem permissão e sem convite

**Comment by janacris** [comment]
@juliakarla Entendo, mas a estrelinha(ou star) pelo que entendi, só serve para adicionar o repositório a sua lista de favoritos, na verdade(para facilitar o acesso depois) .Já o  Pull Request(ou solicitação de pull)que é o que todo mundo está tendo dificuldade aqui)permite que você informe outras pessoas sobre as alterações por push feitas em uma branch de um repositório(achei isso no Google, rs) se tiver errado o @wesleygomes022 pode me corrigir depois(rs).

O ChatGpt também fornece algumas orientações de onde pode estar o possível erro(peguei um aleatório aqui só como exemplo):

![image](https://github.com/GH4WOMAN-M1/GH4Woman-modulo1/assets/89252310/3d263d00-a96b-448b-9910-12e48093f47c)
 
 Espero que ajude, boa noite!

**Comment by lanvsc** [comment]
Pessoal, vá na caixa de mensagens do seu perfil no Github que o invite é por lá. Aqui recebi o meu agora pela manhã e consegui concluir a tarefa.

**Comment by prinakayama** [comment]
Não ha nada na minha caixa de mensagens, não aceitaram o pull request, não responderam nada.

**Comment by janacris** [comment]
Pra quem ainda não recebeu o pessoal do suporte do Programa acabou de enviar um e-mail com a atualização da atividade.Disseram que fizeram uma atualização na Plataforma para quem não conseguiu fazer ainda, poder fazer. Segue:

![image](https://github.com/GH4WOMAN-M1/GH4Woman-modulo1/assets/89252310/c09e21b2-1d3f-47f7-b01e-025b7dc1c837)

**Comment by geovanams** [comment]
Olá meninas!!

Muito obrigada por sinalizarem a questão em relação aos acessos para realização da Atividade do Módulo 1. Estamos adicionando vocês como colaboradoras do repositório, porém existe uma limitação de colaboradores que podem ser adicionados diaramente. Então, foi realizada uma alteração na atividade para que todas possam trabalhar em cima de um repositório através de fork. Não será necessário realizar pull request no momento, pois veremos mais adiante nas aulas 😸. 

Para aquelas  que já possuem acesso e finalizaram a atividade clonando diretamente sem usar o Fork e já subiram a atividade na plataforma, não é necessário realizar novamente!

Obrigada pela compreensão e nos vemos na aula ❤️ ❤️

**Comment by DaianeCP** [comment]
A orientação não ficou clara, devemos aguardar ou fazer pela nova forma? Qual prazo da atividade?

**Comment by lanvsc** [comment]
> A orientação não ficou clara, devemos aguardar ou fazer pela nova forma? Qual prazo da atividade?

Quem não conseguiu fazer deve seguir o novo passo a passo no site com a atividade: https://www.maismulheres.tech/courses/take/github-4-women/assignments/54617191-atividade-modulo-1

**Comment by adelitab** [comment]
@geovanams: como não sabia que a atividade ficaria aberta até o final das aulas, eu fiquei com medo de perder nota e acabei enviando a atividade demonstrando o erro =/
Eu posso seguir com o fork e vocês conseguiriam visualizar?
Obrigada!
